traffic_portal/build/traffic_portal.spec

```diff
@@ -61,7 +61,7 @@ tar -xzvf $RPM_SOURCE_DIR/traffic_portal-%{version}.tgz
 
 	# creates dynamic json file needed at runtime for traffic portal to display release info
 	VERSION=%{version}-%{build_number}
-	BUILD_DATE=$(date +'%Y-%m-%d %H:%M:%S')
+	BUILD_DATE=$(date +'%Y-%m-%d %H:%M')
```

Comment:
Doesn't that just remove the seconds part of the build date?

Reply:
you are correct @ocket8888 but deleting the seconds also deletes the `OURCE` for some reason. /shrug

not sure the seconds are really needed anyhow. this field is purely for display purposes in TP.

Reply:
Odd, seems like this would have been a problem since the beginning.

Reply:
oh hi @dewrich - it has been. just never much of a priority.

Reply:
I bet `%S` is getting expanded as an `rpmbuild` macro

Reply:
probably something like that